ansible/roles/ocp-workload-starter-workshop/defaults/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# PROVIDED BY THE INFRA
+# Uncomment and set a value to take effect. Else, will use the defaults
+#
+# ocp_apps_domain:
+admin_project: 
+user_count_ini: 1
+user_count_end: 50
+user_format: user%02d
```

Comment:
why do you want to use `user%02d` ? why not be backward compatible with what instructors are used to (user1, user2, ..)

why not `user%d` ?